component/http/middleware_test.go

```diff
@@ -551,3 +551,41 @@ func TestAddWithWeight(t *testing.T) {
 		})
 	}
 }
+
+func TestIsConnectionReset(t *testing.T) {
+	tests := []struct {
```

Comment:
We would like all of our table driven tests to be based on maps moving forward. Check out this article for details. https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

Reply:
Yup, easy. https://github.com/beatlabs/patron/pull/334/commits/91731d0cc7fe927c6c5ccd447846a02af004e67b